pyanaconda/ui/tui/spokes/performance.py

```diff
@@ -0,0 +1,91 @@
+# Performance text spoke
+#
+# Copyright (C) 2018  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Martin Kolman <mkolman@redhat.com>
+#
+
+
+from pyanaconda import iutil
+
+from pyanaconda.flags import flags
+from pyanaconda.i18n import N_, _
+
+from pyanaconda.ui.categories.system import SystemCategory
+from pyanaconda.ui.tui.spokes import EditTUISpoke
+from pyanaconda.ui.tui.spokes import EditTUISpokeEntry as Entry
+
+__all__ = ["PerformanceSpoke"]
+
+class PerformanceSpoke(EditTUISpoke):
+    title = N_("Kernel & performance")
+    category = SystemCategory
+
+    edit_fields = [
+        Entry("PTI (Page Table Isolation)", "pti", EditTUISpoke.CHECK, True),
+        Entry("IBRS (Indirect Branch Restricted Speculation)", "ibrs", EditTUISpoke.CHECK, True),
+        Entry("IBPB (Indirect Branch Prediction Barriers)", "ibpb", EditTUISpoke.CHECK, True),
+        ]
+
+    def __init__(self, app, data, storage, payload, instclass):
+        EditTUISpoke.__init__(self, app, data, storage, payload, instclass, "user")
```

Comment:
Is the "user" okay here?

Reply:
Good catch - looks like one can specify the password checking policy like this (kinda strange to have this as part of EditTUISpoke) and the value can be safely left out.